src/Migrators/Migrators.MSSQL/Migrations/Application/20211018230457_InitialCreate.cs

```diff
@@ -137,6 +141,12 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         .Annotation("SqlServer:Identity", "1, 1"),
                     Description = table.Column<string>(type: "nvarchar(max)", nullable: true),
                     TenantKey = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    Group = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    CreatedBy = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    CreatedOn = table.Column<DateTime>(type: "datetime2", nullable: false),
+                    LastModifiedBy = table.Column<string>(type: "nvarchar(max)", nullable: true),
+                    LastModifiedOn = table.Column<DateTime>(type: "datetime2", nullable: true),
+                    RoleId1 = table.Column<string>(type: "nvarchar(450)", nullable: true),
```

Comment:
Can you please elaborate on the reason for adding this new column in addition to an already existing (non-nullable FK) 'RoleId'?

Reply:
Something is definitely broken in identity here... need to check base entity.